tests/unit/completion/test_completionwidget.py

```diff
@@ -145,13 +145,3 @@ def test_completion_item_focus(tree, count, expected, completionview):
         completionview.completion_item_focus(direction)
     idx = completionview.selectionModel().currentIndex()
     assert filtermodel.data(idx) == expected
-
-
-def test_completion_item_focus_no_model(completionview):
```

Comment:
Why remove this? It seems like we should keep this around as a regression test, unless we can guarantee this will never be called without a model set (does your new code guarantee that?)